code/modules/mob/dead/observer/observer.dm

```diff
@@ -815,9 +815,10 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 			if(new_xeno)
 				SSticker.mode.transfer_xeno(src, new_xeno)
 		if("Larva")
-			if(!SSticker.mode.attempt_to_join_as_larva(src))
+			var/mob/living/carbon/xenomorph/mother = SSticker.mode.attempt_to_join_as_larva(src)
+			if(!mother)
 				return
-			SSticker.mode.spawn_larva(src)
```

Comment:
shitcode here

Reply:
Your disagreement is not on code quality, but on design.
This kind of comment really helps nothing.
Try being productive in your review, please.